guides/v2.0/frontend-dev-guide/templates/template-security.md

```diff
@@ -31,7 +31,7 @@ The following code sample illustrates the XSS-safe output in templates:
 <?php echo $block->getTitleHtml() ?>
 <?php echo $block->getHtmlTitle() ?>
 <?php echo $block->escapeHtml($block->getTitle()) ?>
```

Comment:
Also make these changes on versions 2.1, 2.2, and 2.3 if needed.

Reply:
@shrielenee I've checked those changes not required in 2.1, 2.2, and 2.3

Reply:
@shrielenee I've checked those changes not required in 2.1, 2.2, and 2.3